configure.ac

```diff
@@ -209,11 +209,7 @@ PKG_CHECK_MODULES([SSSNSSIDMAP], [sss_nss_idmap >= 1.13.90])
 dnl ---------------------------------------------------------------------------
 dnl - Check for POPT
 dnl ---------------------------------------------------------------------------
-
-POPT_LIBS=
-AC_CHECK_HEADER(popt.h, [], [AC_MSG_ERROR([popt.h not found])])
-AC_CHECK_LIB(popt, poptGetContext, [POPT_LIBS="-lpopt"])
-AC_SUBST(POPT_LIBS)
+PKG_CHECK_MODULES([POPT], [popt])
```

Comment:
Older version of libpopt does not provide pkg-config file.
If you want to use PKG_CHECK_MODULES anyway then it would be good to fall back to older style of detection.
```
sh$ rpm -q popt-devel
popt-devel-1.13-16.el7.x86_64
sh$ pkg-config --list-all | grep pop
```